tests/functional/builtins/codegen/test_slice.py

```diff
@@ -166,7 +166,9 @@ def do_slice(inp: Bytes[{length_bound}], start: uint256, length: uint256) -> Byt
     """
 
     def _get_contract():
-        return get_contract(code, bytesdata, override_opt_level=opt_level)
+        if "__init__" in code:
```

Comment:
why is this branch required?

Reply:
```suggestion
        if "__init__" in code:
            # eth-tester used to ignore constructor arguments if no constructor was defined
            # now we raise an exception, so only call the constructor if it exists
```

Reply:
seems fine, i'd like to refactor so we don't rely on searching the source code but we can add as a note to follow up